@uportal/esco-content-menu/src/components/Ellipsis.vue

```diff
@@ -57,11 +75,9 @@ export default {
         }
 
         while (stNodeHeight > maxHeight) {
-          const newHtml = stNode.innerHTML.substring(
-              0,
-              stNode.innerHTML.trimRight().length - 1
-          );
-          stNode.innerHTML = newHtml.trim();
+          stNode.innerHTML = stNode.innerHTML
+              .substring(0, stNode.innerHTML.trimRight().length - 1)
+              .trim();
```

Comment:
:+1: :art: nice stylistic change here :man_artist:

Reply:
sometimes doing an other thing and getting back permit to have a better review of our code :eyeglasses: